apps/console/src/features/applications/components/settings/sign-on-methods/script-based-flow/template-description.tsx

```diff
@@ -111,6 +112,39 @@ export const TemplateDescription: FunctionComponent<TemplateDescriptionPropsInte
         );
 
     };
+
+    const getParameters = () : string[] => {
+        const params = [];
+        if (isObject(template.parametersDescription)) {
+            Object.entries(template.parametersDescription).map(([param]) => {
+                params.push(param);
+            });
+        }
+        return params;
+    }
+
+    const generatePrerequisite = (prerequisite: string, params: Array<string>): ReactElement => {
+        const sentenceArray = prerequisite.split(" ");
+        const modified = [];
+        let content = ""
+        sentenceArray.map((word, index) => {
```

Comment:
Let's define the types of word and index

Reply:
Resolved in [0aad4b3](https://github.com/wso2/identity-apps/pull/3777/commits/0aad4b38955f51616d9d2927bdaf9c64a2dbfe54)